tests/output_filters/run_filter_test.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+from contextlib import nested
+from StringIO import StringIO
+
+import simplejson
+import testify as T
+from mock import patch
+from osxcollector.output_filters.base_filters.output_filter import run_filter
+
+
+class RunFilterTest(T.TestCase):
+
+    @T.setup_teardown
+    def never_write_api_cache(self):
+        with patch('osxcollector.output_filters.util.api_cache.ApiCache._write_cache_to_file'):
+            yield
+
+    def run_test(self, create_filter, input_blobs=None, expected_output_blobs=None):
+        """Mocks out stdin, stdout, and config then runs input lines through an OutputFilter.
+
+        Args:
+            create_filter: A callable that returns an OutputFilter.
+            input_blobs: An array of dicts to pass to OutputFilter. These will be serialized into strings and passed as stdin.
+            expected_output_blobs: An array of dicts the output of the OutputFilter must match.
+            config_initial_contents: A dict of config.
```

Comment:
The last three of these Args in the comments are no longer arguments for this method.